Kconfig.zephyr

```diff
@@ -49,6 +49,9 @@ menu "Build and Link Features"
 
 menu "Linker Options"
 
+config FAIL_this
```

Comment:
why do we need this?

Reply:
we don't.
Test code that should not have been commit'ed, just to ensure the fix.

Removed now.